ci/travis.sh

```diff
@@ -0,0 +1,25 @@
+#! /bin/bash
+
+# Colors:
+# https://linuxhint.com/tput-printf-and-shell-expansions-how-to-create-awesome-outputs-with-bash-scripts/
+# https://unix.stackexchange.com/questions/269077/tput-setaf-color-table-how-to-determine-color-codes
+
+fgCyan=$(tput setaf 6)
```

Comment:
2019 like it was 1970s

Reply:
:smile: I think that's the first time I see this done properly.
I'm frequently using the ECMA-48 sequences directly as they are documented in the man page `console_codes` but I never used `tput` despite knowing it was around.